src/ImageSequencer.js

```diff
@@ -198,6 +198,28 @@ ImageSequencer = function ImageSequencer(options) {
     return modulesdata;
   }
   
+  function toString(step) {
+    if(step) {
+      return stepToString(step);
+    } else {
+      return  copy(this.images.image1.steps).map(function forEachStep(step){
```

Comment:
@jywarren Actually I did it this way so that toString can be called on a module as well, like sequencer.toString('module'), do you want me to remove that?

Reply:
Oh, hmm, so would that then loop back up to recurse, then get flagged on line 202's conditional, and ... sorry, I guess I don't follow. I guess I'm also interested in this just being a bit more readable if there's other logic going on here. Like, how would `.toString('module')` run differently - won't it  be passed directly to `return toString(step)` either way? Sorry if I'm being dense!!!